cards/cards.css

```diff
@@ -0,0 +1,127 @@
+@import "../for-import/reset.css"; 
+@import "../for-import/colors.css";
+@import "../for-import/temp-buttons.css";
+@import "../for-import/typography.css";
+
+/* style for icon on top of image */
+.card-top {
+    position: relative;
+}
+.icon-on-image {
+    position: absolute;
+    top: 0.5rem;
+    left: 1rem; 
+   color: var(--accent-color) ;
+    font-size: 1.5rem;
+    cursor: pointer
+}
+
+.icon-on-image:hover {
+  color: var(--primary-color);    
+}
+
+
+.dismiss-on-card {
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem; 
+    font-size: 1.3rem;
+    color: var(--primary-color);
+    cursor: pointer
+
+}
+
+.text-on-image {
+    position: absolute;
+    top: 0.5rem;
+    right: 50%; 
+    color: var(--primary-color);
+    font-weight: 700;   
+}
+
+.heading-text-on-image {
+    margin-top: 2rem; 
+}
+
+/* cards style */
+
+.cards {    
+    width: 21.8rem; /* 350px */
```

Comment:
comment /* 350px */ can be removed. If you feel you need for reference, you state why its 350px

Reply:
removed :)